Support/Scripts/ds2-gdb-wrapper.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
```

Comment:
Also I think this is better or I think match other script.
  -  #!/usr/bin/env bash